docfx_yaml/extension.py

```diff
@@ -141,6 +156,14 @@ def _create_datam(app, cls, module, name, _type, obj, lines=None):
         # Make relative
         path = path.replace('/', '', 1)
         start_line = inspect.getsourcelines(obj)[1]
+
+        # append relative path defined in conf.py (in case of "binding python" project)
+        try:
+            source_prefix  # does source_prefix exist in the current namespace
```

Comment:
>source_prefix [](start = 12, length = 13)

Does this needed for other repos? If azure-python-sdk needn't this, it needn't be compulsive.